starport/pkg/cosmoscoin/coin.go

```diff
@@ -20,20 +20,34 @@ var (
 	parseRe     = regexp.MustCompile(pattern)
 )
 
-// Parse parses a coin into amount and denom.
-func Parse(c string) (amount uint64, denom string, err error) {
+// Coin represents a SDK coin
```

Comment:
We started `cosmoscoin` because the idea was to never import SDK into Starport to prevent version anomalies. Now, we already -needed to- import SDK so, I think this package can be removed and we can use `sdktypes.Coin` directly.

Reply:
Mmh ok, given the regex https://github.com/tendermint/starport/blob/b771cbbf305ebecebf0de8f71178895a1ffcfc86/starport/pkg/cosmoscoin/coin.go#L25 I though this package brought more flexibility than the SDK package